backend/fms_core/serializers.py

```diff
@@ -696,4 +715,11 @@ class Meta:
 class StudySerializer(serializers.ModelSerializer):
     class Meta:
         model = Study
-        fields = ("id", "letter", "project_id", "workflow_id", "start", "end")
+        fields = ("id", "letter", "project_id", "workflow_id", "start", "end", "reference_genome_id")
```

Comment:
nope ...you must have merged an old version of master in ...

Reply:
True.

Reply:
It was not an old version. There was a conflict and I chose the wrong line.